platform-support.md

```diff
@@ -0,0 +1,53 @@
+# Platform Support Lifecycle
+
+"Platform Support" for this project includes three somewhat different things:
+1. Test assembly target platforms, that is, the target platforms for which tests may be written.
+2. The target platforms under which those tests will actually run, which is equivalent to the list of agents we provide.
+3. The minimum platforms required to execute the runner and engine themselves, without regard to the tests being run.
+
+## Test Assembly Target Runtimes
+
+We currently support execution of tests written to target any version of the .NET Framework >= 2.0 and any version
+of .NET Core >= 3.1, including .NET 5.0 and higher. We will continue to support tests targeting runtimes which are
+out of support from Microsoft's perspective, so long as we are able to do so without significant additional effort
+and without security risk. If no agent is available for a runtime, the tests will be run on the closest higher 
+runtime available.
+
+## Agents Provided
+
+We currently supply a fairly large selection of agents with the console runner:
+* .NET Framework 2.0
+* .NET Framework 4.6.2
+* ,NET Core 3.1
+* .NET 5.0
+* .NET 6.0
+* .NET 7.0
+* .NET 8.0 (coming in version 3.18.0)
+
+As a general policy, we will continue to provide agents for any Microsoft runtime for at least six months after its
+official end of life. This is intended to support continued testing of legacy applications while users are in the 
+process of upgrade. However, agents for runtimes which have been declared a security risk may be removed immediately.
+
+Based on that policy and the planned end-of-life dates for runtimes, we expect to retire agents on or after the
+dates listed in the following table.
+
+| Runtime              | Agent Supported<br>At Least Until |
+| -------------------- | --------------- |
+| .NET Framework 2.0   |                 |
```

Comment:
Should this be supported at all anymore,  support ended in 2011.

Reply:
OK... I revised the table layout substantially and I hope I put in dates that suit. @OsirisTerje I used "after December 2024" rather than "until further notice" since it at least gives a hint that we will reconsider in some time frame. I think the use of "after" tells folks which dates are "soft" and which are "hard".